app/sprinkles/account/src/Controller/AccountController.php

```diff
@@ -658,8 +668,18 @@ public function pageSettings(Request $request, Response $response, $args)
         // Get a list of all locales
         $locales = $config->getDefined('site.locales.available');
 
+        // Hide the locale field if there is only 1 locale available
+        $fields = [
+            'hidden'   => [],
+            'disabled' => []
+        ];
+        if (count($config->getDefined('site.locales.available')) <= 1) {
```

Comment:
Could reuse `$locales` here. Could be mirrored on line 519 for consistency.

Reply:
Would it be more preferable to use: 
```php
        $localePathBuilder = $this->ci->localePathBuilder;

        // Get locale information
        $currentLocales = $localePathBuilder->getLocales();
```
vs
```php
        // Get a list of all locales
        $locales = $config->getDefined('site.locales.available');
```

in all cases?

Reply:
I'm not 100% sure, but I think `localePathBuilder` will only return the locale associated with the one currently in use? It will at least return them in the inheritance order.

Reply:
In any case, this behavior will be changed in #850, so I guess it doesn't matters for now.